hasher-matcher-actioner/setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from setuptools import setup
+
+
+setup(
```

Comment:
do we know how this will interact with `ThreatExchange/hasher-matcher-actioner/requirements-dev.txt` ?

Reply:
I figured it would work the same way it does now, as additional requirements. The test action seems to be running correctly.